src/repository/projects.rs

```diff
@@ -47,7 +47,7 @@ impl ProjectRepository for PgProjectRepository {
     async fn get(&self, id: i32) -> Result<Vec<Project>, ProjectGetError> {
         let projects = sqlx::query_as::<_, Project>(
             "
-            SELECT project_id, main_document_id, owner, name
+            SELECT project_id, name, created_at, last_modified, owner
```

Comment:
Owner is the user id, rigth? I believe it'll be better to return user name instead

Reply:
should work now